distribution/docker/etc/org.codice.ddf.security.idp.server.IdpEndpoint.config

```diff
@@ -2,6 +2,6 @@ expirationTime=I"30"
xxymfvCduOSDKZh3DJN65oL1PKc8gozZOR978VaxizPCjJNUTCzXQ4pfyHE0GIOkynZT63ZoJhlbJaD7sosbcXdD18CZZ/c3lS+4W2XeMClQoHNgmho14i9I/tf2DJ5j/dIrOk5UJ4mg9xTA3TFnLPLpPiMtvW/QQmfK1z59LBIFnGr9OchiWyJPxe8ND0LEsDv0NQ0CP+LhdM3wNuZnxjV5IdPXp1GKF24ouBUa3kGtYrdu+nj2icpp/tVjOGFnBuswH7rckKEzcmYAdW1buG00AmydLrg8CAwEAAaMXMBUwEwYDVR0RBAwwCoIIc2FtbGhvc3QwDQYJKoZIhvcNAQELBQADgYEAoPIq6VIT/pR0VF2bejTog3JEM4y01qI30CXjP58PVHjPJcQxDSQCQc/GFPWfFNaSwg9LR0nPXbhRPnwpveROEzYXabV60e+8Ny9VsGu7Siu7cdcuM5ZpTdSamtioHZ1LDw4lvtLgXTiYmipm55SY9DIhxVgULylq1tOSblgS66M\=\ \ \ \ \ \ \ \ \ \ \ </ds:X509Certificate>\ \ \ \ \ \ \ \ \ </ds:X509Data>\ \ \ \ \ \ \ </ds:KeyInfo>\ \ \ \ \ </md:KeyDescriptor>\ \ \ \ \ <md:KeyDescriptor\ use\=\"encryption\">\ \ \ \ \ \ \ <ds:KeyInfo\ xmlns:ds\=\"http://www.w3.org/2000/09/xmldsig#\">\ \ \ \ \ \ \ \ \ <ds:X509Data>\ \ \ \ \ \ \ \ \ \ \ <ds:X509Certificate>\ \ \ \ \ \ \ \ \ \ \ \ \ MIICsDCCAhmgAwIBAgIGAWH8X/0MMA0GCSqGSIb3DQEBCwUAMIGEMQswCQYDVQQGEwJVUzELMAkGA1UECBMCQVoxDDAKBgNVBAoTA0RERjEMMAoGA1UECxMDRGV2MRkwFwYDVQQDExBEREYgRGVtbyBSb290IENBMTEwLwYJKoZIhvcNAQkBFiJlbWFpbEFkZHJlc3M9ZGRmcm9vdGNhQGV4YW1wbGUub3JnMCAXDTE4MDMwNTE3MzMyNFoYDzIxMTgwMzA1MTczMzI0WjATMREwDwYDVQQDDAhzYW1saG9zdDCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAKvxMUgWYH/1BBbS6gAV8zmMgqrhlnNvbn8nFyHewcNC4lkYmh3TUXbxxymfvCduOSDKZh3DJN65oL1PKc8gozZOR978VaxizPCjJNUTCzXQ4pfyHE0GIOkynZT63ZoJhlbJaD7sosbcXdD18CZZ/c3lS+4W2XeMClQoHNgmho14i9I/tf2DJ5j/dIrOk5UJ4mg9xTA3TFnLPLpPiMtvW/QQmfK1z59LBIFnGr9OchiWyJPxe8ND0LEsDv0NQ0CP+LhdM3wNuZnxjV5IdPXp1GKF24ouBUa3kGtYrdu+nj2icpp/tVjOGFnBuswH7rckKEzcmYAdW1buG00AmydLrg8CAwEAAaMXMBUwEwYDVR0RBAwwCoIIc2FtbGhvc3QwDQYJKoZIhvcNAQELBQADgYEAoPIq6VIT/pR0VF2bejTog3JEM4y01qI30CXjP58PVHjPJcQxDSQCQc/GFPWfFNaSwg9LR0nPXbhRPnwpveROEzYXabV60e+8Ny9VsGu7Siu7cdcuM5ZpTdSamtioHZ1LDw4lvtLgXTiYmipm55SY9DIhxVgULylq1tOSblgS66M\=\ \ \ \ \ \ \ \ \ \ \ </ds:X509Certificate>\ \ \ \ \ \ \ \ \ </ds:X509Data>\ \ \ \ \ \ \ </ds:KeyInfo>\ \ \ \ \ </md:KeyDescriptor>\ \ \ \ \ <md:SingleLogoutService\ Binding\=\"urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect\"\ Location\=\"https://samlhost:8993/services/saml/logout/redirect\"/>\ \ \ \ \ <md:SingleLogoutService\ Binding\=\"urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST\"\ Location\=\"https://samlhost:8993/services/saml/logout/post\"/>\ \ \ \ \ <md:SingleLogoutService\ Binding\=\"urn:oasis:names:tc:SAML:2.0:bindings:SOAP\"\ Location\=\"https://samlhost:8993/services/saml/logout/soap\"/>\ \ \ \ \ <md:AssertionConsumerService\ Binding\=\"urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect\"\ Location\=\"https://samlhost:8993/services/saml/sso/redirect\"\ index\=\"0\"/>\ \ \ \ \ <md:AssertionConsumerService\ Binding\=\"urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST\"\ Location\=\"https://samlhost:8993/services/saml/sso/post\"\ index\=\"1\"/>\ \ \ \ \ <md:AssertionConsumerService\ Binding\=\"urn:oasis:names:tc:SAML:2.0:bindings:PAOS\"\ Location\=\"https://samlhost:8993/services/saml/sso/paos\"\ index\=\"2\"/>\ \ \ </md:SPSSODescriptor>\ </md:EntityDescriptor>", \
```

Comment:
❓ Does this need to be on one line? I know it doesn't really matter but it's _really_ long.

Reply:
I just copied it directly from the etc file. I didn't want to tamper with these config files. 

Is there no problem if I added a new line? Do I have to escape something?